numbat-cli/src/completer.rs

```diff
@@ -73,6 +74,40 @@ impl Completer for NumbatCompleter {
             ));
         }
 
+        // does it look like we're tab-completing a timezone (via the conversion operator)?
```

Comment:
Having tab completion support for time zones is really great. I was wondering if there would be harm in doing this for all strings, not just string after `->`.

Time zones might end up in other places. And we also have `to`, `➞`, … as conversion operators.

Reply:
I've extended this code to recognize the other conversion operators.  I guess we could tab-complete timezones in all strings, but I think I would defer that change until timezones can actually appear in other places